pyvista/plotting/renderer.py

```diff
@@ -2667,7 +2667,7 @@ def remove_actor(self, actor, reset_camera=False, render=True):
         if name is not None:
             self._actors.pop(name, None)
         self.update_bounds_axes()
-        if reset_camera or not self.camera_set and reset_camera is None:
+        if reset_camera or (not self.camera_set and reset_camera is None):
```

Comment:
Re: this thread https://github.com/pyvista/pyvista/pull/6915/files#r1866776558 and this thread https://github.com/pyvista/pyvista/pull/6915/files#r1866776493, I think this may be the correct logic here:

```suggestion
        if reset_camera or not self.camera_set:
```
When removing an actor should we reset the camera if it hasn't been manually set by the user...? I am not sure how the `camera_set` flag works, is it set automatically internally whenever it's set by the renderer or is this strictly controlled by the user (to avoid resetting manual camera changes)